inject/src/main/java/io/micronaut/context/AbstractInitializableBeanDefinition.java

```diff
@@ -2170,7 +2170,7 @@ private <K> K resolveBean(
             }
         } catch (DisabledBeanException e) {
             if (ConditionLog.LOG.isDebugEnabled()) {
-                ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", argument.getTypeName(), e.getMessage());
+                ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", argument.getTypeName(), e.getMessage(), e);
```

Comment:
Since we're rethrowing we likely don't want the stacktrace logged here.
```suggestion
                ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", argument.getTypeName(), e.getMessage());
```